public/editor/scripts/project/project.js

```diff
@@ -237,22 +238,48 @@ define(function(require) {
             return callback(err);
           }
 
-          // Find an HTML file to open in the project, hopefully /index.html
-          var sh = new _fs.Shell();
-          sh.find(getRoot(), {name: "*.html"}, function(err, found) {
+          // Find the index.html file in the project root to open
+          _fs.readdir(getRoot(), function(err, found) {
             if(err) {
               return callback(err);
             }
 
-            // Look for an HTML file to open, ideally index.html
+            // Try to grab the index.html
             var indexPos = 0;
+            var foundIndexHTML = false;
             found.forEach(function(path, idx) {
               if(Path.basename(path) === "index.html") {
```

Comment:
You can skip this check, since `readdir()` will return only basenames without parent dir info.  Also, you don't need both `indexPos` and `foundIndexHTML`.  Just set `indexPos = -1` when you declare it, then you can see if it's greater than `-1` on 257 below.  Better to reduce your state and drop a variable vs. risking a bug by using 2 to do the job of 1.